packages/table/package.json

```diff
@@ -30,19 +30,16 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "slate": "^0.60.0",
-    "slate-react": "^0.60.0"
+    "slate-react": "^0.60.0",
+    "@udecode/plate-common": "2.0.0"
```

Comment:
Why peer dependency?

Reply:
we use it in `dependencies` in core package